ci/yocto-check-layer.sh

```diff
@@ -14,5 +14,7 @@ CMD="$CMD --no-auto"
 CMD="$CMD --dependency `pwd`/poky/meta `pwd`/meta-qcom"
 # Disable automatic testing of dependencies
 CMD="$CMD --no-auto-dependency"
+# Set machines to all machines defined in this BSP layer
+CMD="$CMD --machines $(echo $(find $TOPDIR/conf/machine/ -name *.conf -exec basename {} .conf \; ))"
```

Comment:
With find we should probably also add '-maxdepth 1' here to avoid surprises later on.

Reply:
agreed, and updated.